pkg/apis/cluster/v1alpha2/machine_types.go

```diff
@@ -31,6 +31,9 @@ const (
 
 	// MachineControlPlaneLabelName is the label set on machines part of a control plane.
 	MachineControlPlaneLabelName = "cluster.sigs.k8s.io/control-plane"
+
+	// ExcludeNodeDrainingAnnotation annotation explicitly skips node draining if set
+	ExcludeNodeDrainingAnnotation = "machine.cluster.sigs.k8s.io/exclude-node-draining"
```

Comment:
How about `machine.cluster.sigs.k8s.io/node-drain` and check if the user sets that to `false`?

Reply:
No, you want to opt into skipping draining.  Having a user set it to false would be unintuitive and you would never want to do this.  You definitely want to set it to the opt-in behavior IMO.